include/vapor/DataMgr.h

```diff
@@ -155,10 +155,10 @@ class VDF_API DataMgr : public Wasp::MyBase {
 
     //! \copydoc DC::GetDimension()
     //
-    bool GetDimension(string dimname, DC::Dimension &dimension) const
+    bool GetDimension(string dimname, DC::Dimension &dimension, long ts) const
```

Comment:
Wouldn't it make more sense to give ts a default value (-1) so that we don't have to change all of the users of this class? Ditto for some of the other classes for which at 'ts' argument was added?

Reply:
For most of the functions I changed, I did make the ts argument optional (although by overloading rather than a default value, e.g. your comment above about ts being unused). In the case of this function, I determined it was important for the callee to be explicitly aware of whether they pass in the correct timestep since this function is often used to present information to the user/get information about the datasets outside of the internal systems which would already be aware of whether they need the timestep information. It would therefore make it very easy for end users to call this function without being aware that they might need timestep information which is often the case, now that we've added this capability.

Reply:
okey doke